src/main/python/jdiq2018_effectiveness/performance.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
```

Comment:
By performance you actually mean effectiveness? I like to be clear about the distinction between effectiveness (e.g., AP) vs. efficiency (e.g., latency).